hintman.cabal

```diff
@@ -23,10 +23,12 @@ source-repository head
 library
   hs-source-dirs:      src
   exposed-modules:     Hintman
+                           Hintman.Suggest.TrailingNewline
```

Comment:
I don't think this needs to be exposed.
We should also align all the modules, for consistent formatting.

Reply:
Let's expose every module, it's usually a bad practice to have `other-modules`

Reply:
Regarding the indentation, I looked at https://github.com/kowainik/co-log/blob/master/co-log/co-log.cabal for inspiration.
@ChShersh Is that correct?

Reply:
I guess if we follow that way of doing things, then we'd need an extra level of indentation.

Reply:
@sphaso @cronokirby I like to add indentation to modules inside `.cabal` file to visually separate different components, so that's right :+1: